tools/ci_build/github/pai/migraphx-ci-pipeline-env.Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM rocm/pytorch:rocm5.2.3_ubuntu20.04_py3.7_pytorch_1.12.1
```

Comment:
How is this file used?

Reply:
We manually use it to generate a new docker image with latest ROCm version and push the new docker image to nnxruntimecibuildenvironment.azurecr.io. Migraphx CI pipeline uses the builded docker image directly.

Reply:
Please have a doc:
1. How to generate the image onnxruntimecibuildenvironment.azurecr.io/migraphx-ci-pipeline-env:rocm5.2.3
2. Who will do the job on a regular basis. For example, once a week.

Reply:
The purpose of having build pipelines is to **automate** build process. Why half of the work of this pipeline needs be done manually?

Reply:
Change the usage of Dockerfile. We build the dockerfile as a step of pipeline now, and use the same register container to store docker image as other pipeline does.